content/docs/04-clusters/06-cluster-management/05-backup-restore/01-add-backup-location.md

```diff
@@ -0,0 +1,357 @@
+	|**S3 URL** | URL of the MinIO object storage console. Example: `http://12.123.234.567:0000`|
+	|**Force S3 path style** | To force S3 path style addressing or else the URL will be converted to virtual-hosted style addressing with bucket name appended to the URL. This is an optional setting. |
+
+
+5. Next, provide the MiniIO unique **Access Key** and **Secret Key**. You can procure a unique access key and the corresponding secret key from the MinIO console. Ensure that the MinIO user has sufficient permissions to perform the required bucket operations. You can refer to the [MinIO official documentation](https://min.io/docs/minio/kubernetes/upstream/administration/identity-access-management/minio-user-management.html#access-keys) to know more.
+
+
+6. Click **Create** to finish the location creation wizard. 
+
+<br />	
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Azure" key="azure">
+
+## Prerequisites
+
+* An active Azure cloud account with the following pieces of information noted down:
+  * Tenant Id
+  * Client Id
+  * Subscription Id
+  * Client Secret created
+
+
+* An [Azure storage account](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal) created with the following information to be noted down for Palette use:
+  * Storage Name: Custom name given to the Azure storage created.
+  * Stock-keeping unit
+
+
+* A container to be created in the Azure Storage account
+
+<br /> <br />	
+
+## Add the Backup in Azure Blob
+
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+
+2. Navigate to the **Project Settings** > **Backup Locations** page. 
+
+
+3. Click on the **Add New Backup Location** button. Palette will open a wizard to configure the new backup location.
+
+
+4. Use the following details to configure a backup location in Azure. 
+
+	|**Field**|**Value**|
+	|---|---|
+	|**Location Name**|Name of your choice.|
+	|**Location Provider**|Select Azure from the drop-down field. |
+	|**Container Name:** | Provide the container created in the Azure Storage. |
+	| **Storage Name** | Name of the Azure Storage resource. |
+	| **Stock-Keeping Unit** | Get this information from the Azure Storage resource. |
+	| **Resource Group:** | Provide the Azure Resource Group name. |
+	|**Tenant ID** | Provide the Azure tenant ID.|
+	| **Client ID** | Provide the Azure client ID of the service principal. Ensure the service principal has sufficient permissions to perform the required bucket operations. |
+	| **Subscription ID** |Provide the Azure subscription ID where you have created the Resource Group and the Azure Storage resource.|
+	| **Client Secret** | Provide the Azure client secret for the service principal you created for Palette to assume. |
+
+
+5. Click **Create** to finish the backup location creation wizard.
+
+<br /> 
+
+</Tabs.TabPane>
+
+</Tabs>
+
+
+# Validate
+
+You can follow the steps outlined below to validate adding a backup location.
```

Comment:
```suggestion
Use the following the steps outlined below to validate adding a backup location.
```
Be direct. It makes it easier to follow along. "You can" is not direct language.